src/build-data/policy/bsi.txt

```diff
@@ -134,12 +139,18 @@ salsa20
 #shake_cipher # not recommended, but needed for kyber
 
 # kdf
-hkdf
+#hkdf // needed for tls 1.3
 kdf1
 kdf2
 prf_x942
 sp800_56a
 
+# pbkdf
+bcrypt_pbkdf
+pbkdf2
+pgp_s2k
+scrypt
+
```

Comment:
As long as BSI doesn't explicitly discourage their usage, we should not prohibit them. Just not enable by default. E.g. `pgp_s2k`: I could imagine that the PGP-related BSI project might even depend on it.